tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,625 @@
+    ```
+    resource "helm_release" "nginx_ingress" {
+    name      = "ingress-nginx"
+    namespace = "ingress-nginx"
+
+    create_namespace = true
+
+    repository = "https://kubernetes.github.io/ingress-nginx"
+    chart      = "ingress-nginx"
+
+    values = [
+        <<-EOF
+        controller:
+            replicaCount: 3
+
+            topologySpreadConstraints:
+            - labelSelector:
+                matchLabels:
+                app.kubernetes.io/name: ingress-nginx
+                app.kubernetes.io/instance: ingress-nginx
+                app.kubernetes.io/component: controller
+            topologyKey: topology.kubernetes.io/zone
+            maxSkew: 1
+            whenUnsatisfiable: DoNotSchedule
+
+            service:
+            enabled: false
+            config:
+            location-snippet: |
+                location /up {
+                return 200 'up';
+                }
+        EOF
+    ]
+    }
+    ```
+
+4. Now add Kubernetes services of type `LoadBalancer` to expose your `ingress-nginx` pods. The important part here is that we will use the `service.beta.kubernetes.io/scw-loadbalancer-zone` annotation to create a load balancer per AZ. This annotation is used by the `scaleway-cloud-controller-manager` to create a load balancer in a specific AZ. In the `fr-par` region, load balancers are available in two AZs, so we will have two load balancers. Each load balancer dispatches traffic to the healthy nodes of all the AZs. This means, in case of the loss of one AZ where one of the two load balancers is located, the other load balancer will still be available and will dispatch traffic to the healthy nodes, and their healthy `ingress-nginx pods`, on the other AZs.
+The selector matches the labels set by the Helm chart on the ingress-nginx pods, and the service exposes both HTTP and HTTPS ports. At the end of the resource, we tell Terraform to ignore some labels and an annotation that `scaleway-cloud-controller-manager` set automatically, and that we do not want to override it on the next Terraform runs.
```

Comment:
```suggestion
4. Now add Kubernetes services of type `LoadBalancer` to expose your `ingress-nginx` pods. The important part here is that we will use the `service.beta.kubernetes.io/scw-loadbalancer-zone` annotation to create a load balancer per AZ. This annotation is used by the `scaleway-cloud-controller-manager` to create a load balancer in a specific AZ. In the `fr-par` region, load balancers are available in two AZs, so we will have two load balancers. Each load balancer dispatches traffic to the healthy nodes of all the AZs. This means, in case of the loss of one AZ where one of the two load balancers is located, the other load balancer will still be available and will dispatch traffic to the healthy nodes, and their healthy `ingress-nginx` pods, on the other AZs.
The selector matches the labels set by the Helm chart on the ingress-nginx pods, and the service exposes both HTTP and HTTPS ports. At the end of the resource, we tell Terraform to ignore some labels and annotations that the `scaleway-cloud-controller-manager` sets automatically, and that we do not want to override on the next Terraform runs.
```